kas/demos/meta-mender-raspberrypi-wifi/recipes-raspberrypi/images/core-image-raspberrypi-wifi.bb

```diff
@@ -0,0 +1,10 @@
+SUMMARY = "A console-only image that fully supports the target device \
+hardware."
+
+LICENSE = "MIT"
```

Comment:
Why MIT? `wificonfig` should have its own license, so this should still be Apache, no?

I suppose it does not matter that much, since this does not install any files that aren't already license tracked elsewhere, but for completeness' sake.

Reply:
@kacf that snuck in by starting out from `core-image-minimal`, but it seems that an image actually doesn't need a license. So I'll just remove it.